hatch/src/assets/tests/builder.rs

```diff
@@ -1,7 +1,7 @@
-use assets::builder::Builder as AssetBuilder;
-use assets::ProjectAsset;
-use project::{ Project, ProjectKind, LibraryKind };
-use std::path::PathBuf;
+#[cfg(test)] use assets::builder::Builder as AssetBuilder;
```

Comment:
After reading this documentation: https://doc.rust-lang.org/book/second-edition/ch11-03-test-organization.html can this annotation simply be added next to the test module definition within the mod.rs file and not on every one of these lines? Or is there another reason they need to be on every use statement?

Reply:
You are absolutely correct!